app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+  <div class="card-body">
+    <h2 class="card-title"><%= @user.full_name %></h2>
+    <a class="card-subtitle mb-2 text-muted" href="mailto:<%= @user.email %>"><%= @user.email %></a>
+    
+    
+    <% if @user.description.present? %>
+        <hr>
```

Comment:
Nitpicky suggestion: instead of using a `<hr>` it might be preferable to have two `.card-body`s next to each other with a CSS top border, something like this:

```
<div class="card">
    <div class="card-body"><!-- meta info --></div>
    <div class="card-body border-top"><!-- description --></div>
</div>
```

Reply:
Let's stick with the `<hr>`, if we find problems with it in the future we'll resort to two `.card-body` elements.